contracts/vaults/Claim.sol

```diff
@@ -62,14 +63,27 @@ contract Claim is Base {
         );
     }
 
+    function challengeClaim(uint256 _claimId) external onlyArbitrator {
+        require(claims[_claimId].beneficiary != address(0), "HVE10");
+        claims[_claimId].isChallenged = true;
```

Comment:
I think we should also check  that the claim hasn't been challenged already

Reply:
we could but why

Reply:
To prevent the connector challenging it a second time. On the other hand we can do the same check on the connector's side, so you're probably right. There is no need